package.json

```diff
@@ -46,10 +46,9 @@
     },
     "repository": {
         "type": "git",
-        "url": "git+https://github.com/georgejecook/roku-asset-bundler/Roku-asset-bundler.git"
+        "url": "https://github.com/georgejecook/roku-asset-bundler.git"
     },
     "keywords": [
-        "ropm",
```

Comment:
This is the most important part. This is not a ropm module, so it shouldn't be tagged as such.